src/components/AddTodo/AddTodoForm.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import './AddTodoForm.css';
+import PropTypes from 'prop-types';
+
+export class AddTodoForm extends React.Component {
+  state = {
+    authors: this.props.authors
+      .concat({
+        id: Math.random(),
+        name: 'Choose an author',
+      })
+      .sort((author1, author2) => author1.id - author2.id),
```

Comment:
you probably don't need derived state
https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html